src/components/Header/ChainConnectivityWarning.tsx

```diff
@@ -0,0 +1,72 @@
+import { Trans } from '@lingui/macro'
+import { CHAIN_INFO, L2ChainInfo } from 'constants/chains'
+import { useActiveWeb3React } from 'hooks/web3'
+import { AlertOctagon } from 'react-feather'
+import styled from 'styled-components/macro'
+import { ExternalLink, MEDIA_WIDTHS } from 'theme'
+
+const BodyRow = styled.div`
+  color: black;
+  font-size: 12px;
+`
+const CautionIcon = styled(AlertOctagon)`
+  color: black;
+`
+const Link = styled(ExternalLink)`
+  color: black;
+  text-decoration: underline;
+`
+const TitleRow = styled.div`
+  align-items: center;
+  display: flex;
+  justify-content: flex-start;
+  margin-bottom: 8px;
+`
+const TitleText = styled.div`
+  color: black;
+  font-weight: 600;
+  font-size: 16px;
+  line-height: 20px;
+  margin: 0px 12px;
+`
+const Wrapper = styled.div`
+  background-color: ${({ theme }) => theme.yellow3};
+  border-radius: 12px;
+  bottom: 60px;
+  display: none;
+  max-width: 348px;
+  padding: 16px 20px;
+  position: absolute;
+  right: 16px;
+  @media screen and (min-width: ${MEDIA_WIDTHS.upToMedium}px) {
```

Comment:
```
  ${({ theme }) => theme.mediaWidth.upToMedium`
    display: block;
  `};
```

is probably cleaner

Reply:
I'm intending to use the `min-width` behavior here instead of `max-width` in the theme. In general I think using min-width for query selectors is a better pattern.  Might be worth bringing up in eng review how to approach this. I don't like the pattern divergence, but in my ideal world we'd use `theme.mediaWidth.greaterThanMedium` or something instead.

Reply:
I actually do like the upToMedium better 

@cmcewen mentioned in a chat we had that maybve we should formalize / agree on how we do certain thinsg like this (media queries, layering, even ordering of css properties)

Reply:
I think we should use the theme-defined lambdas for consistency. Can you not use max-width and flip the css inside the block?

Reply:
Yeah we def *can*, but going smaller→larger allows for progressive overwrites without requiring "erasing" old values that aren't relevant in the next larger screen context. The need to set css back to default values leads to more bugs in my experience (and more code).

Reply:
There are obv exceptions, but I think the mobile-first (min-width) pattern often ends up being cleaner.